src/client/src/containers/SelectFrameworks/index.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { connect } from "react-redux";
+
+import styles from "./styles.module.css";
+
+import SelectBackEndFramework from "../SelectBackendFramework";
+import SelectFrontEndFramework from "../SelectFrontEndFramework";
+import { getVSCodeApiSelector } from "../../selectors/vscodeApiSelector";
+
+import { AppState } from "../../reducers";
+
+import { EXTENSION_MODULES, EXTENSION_COMMANDS } from "../../utils/constants";
+
+interface ISelectFrameworksProps {
+  vscode: any;
```

Comment:
Do you think we could get some type safety here too? 

https://github.com/microsoft/WebTemplateStudio/blob/28759f22376ae8b25401fb8591d13dcb7148d168/src/client/src/reducers/vscodeApiReducer.ts#L19-L21